src/main/java/org/maas/Start.java

```diff
@@ -61,17 +60,27 @@ public static void main(String[] args) {
 
         }
         if(bakingStage) {
-
+			Initializer init = new BakingStageInitializer();
```

Comment:
I am not sure about adding the stages of a specific team to the Start.java. If I am not wrong, in case there is a need to launch some other teams stage we might need to rewrite some parts of this file. 
I haven't spent enough time understanding Start.java yet. Therefore please excuse me if I am completely wrong about this. :)

Reply:
To add the initializer object is the only "proper" way to add agents to start.java. Also, if one wants to not include the agents in baking stage (not that anyone other than right-brothers team can, because the code in not present in this repo), one can not include `-baking` in the `gradle run` command. :slightly_smiling_face: